gator_grouper/settings.py

```diff
@@ -0,0 +1,116 @@
+"""
+Django settings for gator_grouper project.
+
+Generated by 'django-admin startproject' using Django 2.1.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "jc*l8zi5+h1_^f6i)x@0v%l=rw=219!i(dyf@2%_ez1e6mta$x"
```

Comment:
Very bad thing here!!!!

This needs to be taken care of and possibly changed since we absolutely *cannot* publish a secret key.

If this can be regenerated (@sutterj?), then we should merge this and then create a new PR to handle this issue. If not, other methods should be looked into.

Reply:
http://fearofcode.github.io/blog/2013/01/15/how-to-scrub-sensitive-information-from-django-settings-dot-py-files/

I can follow this and change it right now

Reply:
Note that we will still need to regenerate the key, and that this must be done after this merge is completed, according to the [git rewriting instructions](https://help.github.com/en/articles/removing-sensitive-data-from-a-repository).

Reply:
If you are planning on rerunning startapp and startproject in order to rename the directories (without the process of using a Django app) then you will end up getting a new key and could delete the old one.

Reply:
@Michionlion agreed. @barrezuetai  I can help with this.